src/Http/Controllers/VoyagerMediaController.php

```diff
@@ -129,7 +129,7 @@ public function move_file(Request $request)
         $source = "{$location}/{$source}";
         $destination = strpos($destination, '/../') !== false
             ? $this->directory.'/'.dirname($folderLocation).'/'.str_replace('/../', '', $destination)
-            : "{$location}/{$destination}";
+            : "/{$destination}";
```

Comment:
Is the starting `/` necessary? This was a relative path previously.  I'm thinking this could come back and bite us, but I don't fully understand this code right now, so if it's necessary, just say so.

Reply:
I will make more tests, and I will return to you with a better answer.

Reply:
@fletch3555 So, I made some tests and both works fine. So, you decide if you want that I remove the `/`. 

A example of the value of `$destination` variable when I try to move a file:

With slash:`/pages/test/l7a0l6M2OU99o25Ey2kMMGTESVH4spJNA92Y5968.jpeg`

Wihout slash:  `pages/test/l7a0l6M2OU99o25Ey2kMMGTESVH4spJNA92Y5968.jpeg`

Reply:
Another thing: When I try to move a file to a parent folder, the value of `$destination` variable is : 

`/.//l7a0l6M2OU99o25Ey2kMMGTESVH4spJNA92Y5968.jpeg`

Reply:
That's a REALLY weird path when you move the file... Does it still work?

Reply:
About this: `/.//l7a0l6M2OU99o25Ey2kMMGTESVH4spJNA92Y5968.jpeg`? Yes, it's works. Without and with `/`.